Makefile

```
@@ -79,7 +79,7 @@ lint-python:
 lint-html:
 	@yarn htmlhint "testCourse/**/question.html" "exampleCourse/**/question.html" "site"
 lint-markdown:
-	@yarn markdownlint "docs/**/*.md"
+	@yarn markdownlint --ignore node_modules --ignore exampleCourse --ignore testCourse --ignore "**/dist/**" "**/*.md"
```

I have a directory `workspaces/xtermjs/src/node_modules` locally, which produced a ton of false positives.

```suggestion
	@yarn markdownlint --ignore "**/node_modules/**" --ignore exampleCourse --ignore testCourse --ignore "**/dist/**" "**/*.md"
```